src/utils/rfq_blockchain_utils.ts

```diff
@@ -74,4 +75,25 @@ export class RfqBlockchainUtils {
     public generateMetaTransactionCallData(metaTx: MetaTransaction, metaTxSig: Signature): string {
         return this._exchangeProxy.executeMetaTransaction(metaTx, metaTxSig).getABIEncodedTransactionData();
     }
+
+    public getAddressFromIndexAndPhrase(mnemonic: string, index: number): string {
+        const hdNode = HDNode.fromMnemonic(mnemonic).derivePath(this._getPathByIndex(index));
+
+        return hdNode.address;
+    }
+
+    public getPrivateKeyFromIndexAndPhrase(mnemonic: string, index: number): string {
+        const hdNode = HDNode.fromMnemonic(mnemonic).derivePath(this._getPathByIndex(index));
+
+        return hdNode.privateKey;
+    }
+
+    // tslint:disable-next-line:prefer-function-over-method
+    private _getPathByIndex(index: number): string {
+        // ensure index is a 0+ integer
+        if (index < 0 || index !== Math.floor(index)) {
+            throw new Error(`invalid index`);
+        }
+        return `m/44'/60'/0'/0/`.concat(String(index));
+    }
```

Comment:
making a note here to make these functions `static` so they can be used before an instance of `RfqBlockchainUtils` is constructed